c2corg_ui/static/js/map/map.js

```diff
@@ -619,6 +625,66 @@ app.MapController.prototype.handleMapSearchChange_ = function(event) {
 
 
 /**
+ * @param {ol.MapBrowserEvent} event
+ * @private
+ */
+app.MapController.prototype.handleMapFeatureClick_ = function(event) {
+  var feature = this.map.forEachFeatureAtPixel(event.pixel, function(feature) {
+    return feature;
+  }, this, function(layer) {
+    // test only features from the current vector layer
+    return layer === this;
```

Comment:
Good catch! Actually I have hesitated using this syntaxe but I thought passing ``this.getVectorLayer_()`` as the ``this`` argument of the function would make ``getVectorLayer_()`` be called only once, not for every tested layer.
But perhaps it's not that significant (not such a big deal and not that many layers to test) and as you said, it's a bit clearer.
What do you recommend?